### ISSUE TITLE
cannot fail startup by sending an error to done()

### ISSUE THREAD

**Issue by jbcpollak** [enobufs/mocha-prepare#5]
We had a situation where our startup scripts would just hang forever because our startup test looks like something like this:

```
prepare(function(done) {
	app.wfmStarted
	.then(() => done())
	.catch((err) => {
		// eslint-disable-next-line no-console
		console.log(`Unable to initialize due to error`, err);

		done(err);
	});
});
```

Ideally this would fail the prepare and the process would bail out, but this doesn't happen, so instead mocha hangs indefinitely.

**Comment by enobufs** [comment]
That is a good point.

When done() is called with an error object, I could skip the actual Mocha#run() call, and call process.exit(1). Does this sound ok to you?

You could call process.exit(1) on your own inside onPrepare for now.

**Comment by jbcpollak** [comment]
Yeah, that works for me. We call process.exit(1) now manually, but it seems like the sort of thing the library should support.

**Comment by enobufs** [comment]
Published v0.1.0.
